### DISCUSSION TITLE
welcome-to-hnu

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#31] [Announcements]
# /welcome-to-hnu

A magical documentation generator.

https://hainanu-application.github.io/#/welcome-to-hnu

**Comment by junzhang-zj** [comment]
真好，16级密码学来支持下。现在看来只有网安和计院比较多人分享😂

**Reply by peng-yq** [reply]
目前来看是的哈哈哈，还得是计网
